awx/main/scheduler/kubernetes.py

```diff
@@ -0,0 +1,86 @@
+import time
+
+from django.conf import settings
+from kubernetes.client import Configuration as kubeconfig
+from kubernetes.client import CoreV1Api as v1_kube_api
+from kubernetes.client import ApiClient as kube_api_client
+
+# AWX
+from awx.main.utils.common import parse_yaml_or_json
+from awx.main.utils import (
+    create_temporary_fifo,
+)
+
+
+class PodManager(object):
+
+    def __init__(self, task):
+        self.task = task
+        self.credential = task.instance_group.credential
+        self.kube_api = self._kube_api_connection(self.credential)
+        self.pod_name = f"job-{task.id}"
```

Comment:
I'm thinking of the case where users have multiple tower servers, dev and prod, pointed at the same kubernetes cluster. Can we add something unique from the tower cluster to the pod name?